daisy_workflows/image_build/imageimport.wf.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This should go in examples instead of image builds ... or perhaps another directory- image_imports